index.html

```diff
@@ -45,6 +45,15 @@ <h3>Anand Baburajan</h3>
       <a href="https://github.com/anandbaburajan">GitHub</a>
       </p>
       <h3>Fathma Mehnoor</h3>
+      <p>
+        Member at Moonshot, and a B.Tech CSE (S2) student at GEC Palakkad. These days she's learning Python and working on a NFT generative art project.
+      </p>
+      <p>Get in touch:
+      <a href="mailto:fathmamehnoor@gmail.com">Email</a>
+      <a href="https://www.linkedin.com/in/fathma-mehnoor-7b2142229/">LinkedIn</a>
```

Comment:
@fathmamehnoor, just in case you didn't know, you can change your URL on LinkedIn to https://www.linkedin.com/in/fathmamehnoor. Do you wanna do that now (we can wait)? Or should we go ahead and merge this PR?

Reply:
I changed it to https://www.linkedin.com/in/fathmamehnoor. Should I update the PR again?

Reply:
Yep